src/tests.js

```diff
@@ -64,11 +54,18 @@ function Test(component, config) {
     }
   }
 
+  const mixins = config.mixins === undefined ? {} : config.mixins
+
   return testComponent.mixin({
     find: Find,
     setState: SetState,
-    simulate: Simulate
+    simulate: Simulate,
+    ...mixins
   })
 }
 
+export const addMixins = (Test, mixins) => {
+  return (component, config) => Test(component, { ...config, mixins })
+}
```

Comment:
Maybe name it createTest? Hmm I see what you're saying but at the same time
I feel like importing Test and the mixins seems like extra work. Wonder
what everyone else thinks.
On Mon, Oct 5, 2015 at 20:57 Ramana Venkata <notifications@github.com>
wrote:

> In src/tests.js
> <https://github.com/Legitcode/tests/pull/26#discussion_r41223970>:
>
> >    })
> >  }
> >
> > +export const addMixins = (Test, mixins) => {
> > +  return (component, config) => Test(component, { ...config, mixins })
> > +}
>
> I actually considered that. But decided not to do it because of its
> implicit nature. I don't want to make a function impure unless it is
> necessary. I like being explicit. When I read CustomTest =
> addMixins(mixins), I find asking myself addMixins to what. That's my
> reasoning. What do you think??
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Legitcode/tests/pull/26/files#r41223970>.
>